doc/architecture/software-update.rst

```diff
@@ -35,9 +66,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
```

Comment:
Is this true today?  I've read comments that we're not generating this hash currently.